libs/api/domains/national-registry-x-road/src/lib/nationalRegistryXRoad.resolver.ts

```diff
@@ -83,4 +85,32 @@ export class NationalRegistryXRoadResolver {
       person.nationalId,
     )
   }
+
+  @ResolveField('birthplace', () => NationalRegistryBirthplace, {
+    nullable: true,
+  })
+  @Audit()
+  async resolveBirthPlace(
+    @Context('req') { user }: { user: User },
+    @Parent() person: NationalRegistryPerson,
+  ): Promise<NationalRegistryBirthplace | undefined> {
+    return await this.nationalRegistryXRoadService.getBirthplace(
+      user,
+      person.nationalId,
+    )
+  }
+
+  @ResolveField('citizenship', () => NationalRegistryCitizenship, {
+    nullable: true,
+  })
+  @Audit()
+  async resolveCitizenship(
+    @Context('req') { user }: { user: User },
+    @Parent() person: NationalRegistryPerson,
+  ): Promise<NationalRegistryCitizenship | undefined> {
+    return await this.nationalRegistryXRoadService.getCitizenship(
```

Comment:
Ditto on return await.

Reply:
There are return awaits on most of the endpoints in this resolver, I'll remove them all then.